### ISSUE TITLE
EXP: Clustering of low-dim drug embeddings (after drug embedder transformation) `drug_emb_ld_clustering`

### ISSUE THREAD

**Issue by siboehm** [theislab/chemCPA#80]
## Idea: 
Observing how the transformed drug embeddings cluster, and comparing the clustering to Vanilla, to the clustering of Trapnell gene expressions (supp figure in Sciplex paper), to the clustering of untransformed drug embeddings, to the clustering of pretrained-then-finetuned models.

Which precise plots we'll include in the paper is unclear. 
Overall the goal is to give credence to the claim that the chemical embeddings are meaningful and contribute to lowering the CCPA loss.

## Tasks
- [ ] @siboehm  Writes code that can load a CPA model given just the config hash
- [ ] @MxMstrmn Has a look at Oksana's thesis. She made a lot of embedding UMAP plots and some of them may be worth repeating for our setting.
- [ ] @MxMstrmn Performs the clustering, including tuning hyperparameters for UMAP.

Currently we're only planning to perform this experiment for Trapnell, as we don't have information about the drug pathways for LINCS.
